build.gradle.kts

```
@@ -23,7 +23,7 @@ testing {
                 implementation(project())
                 implementation("ej.api:edc:1.3.7")
                 implementation("ej.api:bon:1.4.4")
-                implementation("ej.library.test:junit:1.10.1")
+                implementation("ej.library.test:junit:1.11.0")
```

```suggestion
                implementation("ej.library.test:junit:1.10.3")
```
The version 1.11.0 must be published to central repo to use it in the templates.